cmd/manager/main.go

```diff
@@ -124,12 +128,30 @@ func main() {
 	}
 
 	metricsServer := metrics.NewBuilder().WithPort(metricsPort).WithPath(metricsPath).
+		WithCollectors(localmetrics.MetricPagerDutyCreateFailure).
+		WithCollectors(localmetrics.MetricPagerDutyDeleteFailure).
+		WithCollectors(localmetrics.MetricPagerDutyHeartbeat).
 		GetConfig()
+
 	// Configure metrics if it errors log the error but continue
+
 	if err := metrics.ConfigureMetrics(context.TODO(), *metricsServer); err != nil {
 		log.Error(err, "Failed to configure Metrics")
 	}
 
+	client := mgr.GetClient()
+	pdAPISecret := &corev1.Secret{}
+	err = client.Get(context.TODO(), types.NamespacedName{Namespace: "pagerduty-operator", Name: "pagerduty-api-key"}, pdAPISecret)
+	APIkey := ""
```

Comment:
Defining this as `var APIKey string` is a better syntax for Go.

Reply:
Done